test/main_test.ts

```diff
@@ -83,3 +83,11 @@ class A {
     });
   });
 });
+
+describe('regression', () => {
```

Comment:
Reverted the test file. This is really a test for the library that wraps dart2js output so it can run on NodeJS.